### DISCUSSION TITLE
1.14.0

### DISCUSSION THREAD

**Discussion by JohnnyMorganz** [JohnnyMorganz/luau-lsp#229] [Announcements]
### Added

- Show inlay hints for variables in `for x in y` loops

### Changed

- Sync to upstream Luau 0.553

### Fixed

- Fixed rename symbol not working when cursor after variable
- Fixed rename symbol causing server crashing when attempting to rename a token which is not a variable

<hr /><em>This discussion was created from the release <a href='https://github.com/JohnnyMorganz/luau-lsp/releases/tag/1.14.0'>1.14.0</a>.</em>

**Comment by fewkz** [comment]
AWESOME!!!!!!
